context/myuuid.go

```diff
@@ -11,13 +11,14 @@ import (
 )
 
 var (
-	myUUIDFile = path.Join(config.GetString("localstatedir"), "uuid")
+	myUUIDFile string
 )
 
 // InitMyUUID initializes MyUUID by reading the `<config.LocalStateDir>/uuid` file.
 // If the file is not present it generates a new UUID and saves it to the file.
 // If the file is not accessible. initMyUUID panics.
 func InitMyUUID() uuid.UUID {
+	myUUIDFile = path.Join(config.GetString("localstatedir"), "uuid")
```

Comment:
About `SetDefaults`, I had a comment explaining why I had it. Not all configuration options might have a flag exposed. `SetDefault` could be used for setting the defaults for these config options. But I agree it's useless right now.